packages/react-charting/src/components/SankeyChart/__snapshots__/SankeyChart.test.tsx.snap

```diff
@@ -5326,7 +5704,6 @@ exports[`SankeyChart - mouse events Should render correctly on node mouseover 1`
           outline: none;
         }
     data-focuszone-id="FocusZone52"
-    data-tabster="{\\"uncontrolled\\": {}}"
```

Comment:
You would need to pull master to get this property in your branch

Reply:
I've pulled and re-ran the tests, but they seem to still be removed.

Reply:
So I manually reverted those.